src/hotspot/share/gc/shenandoah/shenandoahMonitoringSupport.cpp

```diff
@@ -36,7 +36,7 @@ class ShenandoahYoungGenerationCounters : public GenerationCounters {
   ShenandoahYoungGenerationCounters() :
           GenerationCounters("Young", 0, 0, 0, (size_t)0, (size_t)0) {};
 
-  void update_all() override {
+  void update_all() {
```

Comment:
Shenandoah looks a bit odd now. @kdnilsen @wkemper and @ysramakrishna may want to take a look?